### ISSUE TITLE
hover on non existing notes works well in notes and on the global graph but not on the local graph

### ISSUE THREAD

**Issue by nicolasmarc1** [nothingislost/obsidian-hover-editor#37]
Hi, minor bug: trying to hover on non existing notes in the local graph doesn't do anything (unlike the global graph)
thanks!

**Comment by nicolasmarc1** [comment]
That's strange. Here's a demo of it working the global graph, but not in the local graph (until i created the file).
This demo was made in another vault with the last version of hover editor and all other community plugins deactivated and nothing came up in the console.

**Comment by nicolasmarc1** [comment]
https://user-images.githubusercontent.com/51007222/158466123-96fb0ab6-0f2c-4386-b398-b3429521e5d6.mp4

here's the demo

**Comment by nothingislost** [comment]
Yeah, it looks like this is because the patching I'm doing is not happening quick enough at application startup in order to patch that local graph view you have in the sidebar. If you open a new local graph pane using the "open local graph" context menu item, it should work then.

This is a bug though and I'll see if I can patch earlier or force the view to be recreated on app startup.

**Comment by nicolasmarc1** [comment]
> Yeah, it looks like this is because the patching I'm doing is not happening quick enough at application startup in order to patch that local graph view you have in the sidebar. If you open a new local graph pane using the "open local graph" context menu item, it should work then.
> 
> This is a bug though and I'll see if I can patch earlier or force the view to be recreated on app startup.

thanks, indeed it works!
